README.md

```diff
@@ -1 +1,128 @@
+  - [🎥 Walkthrough](#walkthrough)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project 1] <a name="about-project"></a>
+
+This project is the capstone project that portrays a conference website.
+
+**[Capstone Project]** is a simple project demonstrating responsive conference website.
+
+## 🚀 VISIT SITE <a name="website"></a>
+
+<a name="<https://king-benx.github.io/my-portfolio/>">https://king-benx.github.io/my-portfolio/</a>
+
+## 🎥 WALKTHROUGH <a name="walkthrough"></a>
+
+<a name="<<https://www.loom.com/share/785d791fbd164d8b9a4ad7f206cc0393/>>">https://www.loom.com/share/785d791fbd164d8b9a4ad7f206cc0393/</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML, CSS AND JAVASCRIPT  
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS & JS</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- JavaScript file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```git clone git@github.com:King-Benx/capstone-project-1.git```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+1. Right click file and run in browser.
+2. Open file in vscode right click and open in live server
+```
+
+## 👥 Author <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Benard Asiimwe**
+
+- GitHub: [@King-Benx](https://github.com/King-Benx)
+- Twitter: [@bataringaya_ben](https://twitter.com/bataringaya_ben)
+- LinkedIn: [Benard Asiimwe Bataringaya](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse learning resources for all the helpful information rendered in completion of this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] **[Optional] Invalid MIT link 😨**
    I think it would be better to provide a  valid **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 

    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._